README.md

```diff
@@ -1,49 +1,61 @@
-# Jujutsu VCS
+<div class="title-block" style="text-align: center;" align="center">
```

Comment:
TBH, the current README.md is, in my opinion, *already* unreadable as pure plain text, primarily due to the usage of inline Markdown links rather than out-of-line ones. That can be fixed separately at least, but IMO for the frontmatter, I think it makes sense to use a little HTML to make it read nicer...